cmd/crio-status/main.go

```diff
@@ -47,27 +49,7 @@ func main() {
 		},
 	}
 	app.Commands = criocli.DefaultCommands
-	app.Commands = append(app.Commands, []*cli.Command{{
-		Action:  config,
-		Aliases: []string{"c"},
-		Name:    "config",
-		Usage:   "Show the configuration of CRI-O as TOML string.",
-	}, {
-		Action:  containers,
-		Aliases: []string{"container", "cs", "s"},
-		Flags: []cli.Flag{&cli.StringFlag{
-			Name:    idArg,
-			Aliases: []string{"i"},
-			Usage:   "the container ID",
-		}},
-		Name:  "containers",
-		Usage: "Display detailed information about the provided container ID.",
-	}, {
-		Action:  info,
-		Aliases: []string{"i"},
-		Name:    "info",
-		Usage:   "Retrieve generic information about CRI-O, like the cgroup and storage driver.",
-	}}...)
+	app.Commands = append(app.Commands, criocli.StatusCommand)
```

Comment:
I believe this results in a behavior change, where the command becomes `crio-status status` unless I'm missing something

Reply:
Fixed. I forgot to add the sub-commands directly to the `crio-status`.